azure/functions/authenticationevents.py

```diff
@@ -1,6 +1,8 @@
 from importlib import import_module
 import json
+from azure.functions import HttpRequest
```

Comment:
Any reason you are importing this, as it is native to the Http Request/Response trigger, and they more than likely have logic in their converter to build this.